drift/lib/src/runtime/query_builder/components/order_by.dart

```diff
@@ -23,9 +37,21 @@ class OrderingTerm extends Component {
   /// The ordering mode (ascending or descending).
   final OrderingMode mode;
 
+  /// How to order NULLs.
+  /// When [nullsOrder] is [null], then it's ignored.
+  ///
+  /// Note that this feature are only available in sqlite3 version `3.30.0` and
+  /// newer. When using `sqlite3_flutter_libs` or a web database, this is not
+  /// a problem.
+  final NullsOrder? nullsOrder;
+
   /// Creates an ordering term by the [expression] and the [mode] (defaults to
   /// ascending).
-  OrderingTerm({required this.expression, this.mode = OrderingMode.asc});
+  OrderingTerm({
+    required this.expression,
+    this.mode = OrderingMode.asc,
+    this.nullsOrder,
+  });
```

Comment:
The name of the field is fine, but I think it would read nicer if the name of the parameter was `nulls`:

```suggestion
    NullsOrder? nulls,
  }): nullsOrder = nulls;
```

That way, users can write `OrderingTerm(..., nulls: NullsOrder.last)` which is a bit closer to the original SQL :)